test/cctest/test_url.cc

```diff
@@ -109,3 +109,36 @@ TEST_F(URLTest, ToFilePath) {
 
 #undef T
 }
+
+TEST_F(URLTest, FromFilePath) {
+  URL file_url;
+#ifdef _WIN32
+  file_url = URL::FromFilePath("C:\\Program Files\\");
+  EXPECT_EQ(file_url.protocol(), "file:");
```

Comment:
Nit: swap the expected/actual. gtest uses a different order from Node.js' assert.

Reply:
done.